crates/driver_net/src/lib.rs

```diff
@@ -3,7 +3,14 @@
 #![no_std]
 #![feature(const_mut_refs)]
 #![feature(const_slice_from_raw_parts_mut)]
+#![feature(box_into_inner)]
 
+#[macro_use]
+extern crate log;
```

Comment:
unused `#[macro_use]` import

Reply:
It' better to remove `extern crate log` here, ad call `log::error!` in `axgbe.rs`.